migrations/versions/917195d9b5e9_add_messageembedding_table.py

```
@@ -0,0 +1,117 @@
+Create Date: 2024-01-01 12:00:00.000000
+
+"""
+
+from collections.abc import Sequence
+
+import sqlalchemy as sa
+from alembic import op
+from pgvector.sqlalchemy import Vector
+
+from migrations.utils import index_exists, table_exists
+from src.config import settings
```

Pylint (3.3.7)

[error] Unable to import 'src.config'

(E0401)